packages/@react-spectrum/label/chromatic/Label.chromatic.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import {ComponentMeta, ComponentStoryObj} from '@storybook/react';
+import {Label} from '../';
+import React from 'react';
+import {SpectrumLabelProps} from '@react-types/label';
+import {TextField} from '@react-spectrum/textfield';
+
+type LabelStory = ComponentStoryObj<typeof Label>;
+
+export default {
+  title: 'Label',
+  component: Label,
+  args: {width: '100%'}
+} as ComponentMeta<typeof Label>;
+
+export let Default: LabelStory = {
+  render: (args) => <TextFieldLabel {...args} />
```

Comment:
We can do a similar setup to the helptext so we don't need a custom render, we can use a decorator for these so args is just applied directly to the `Label`